source/administration/changelog.md

```diff
@@ -200,6 +200,44 @@ The following config settings will only work on servers with an Enterprise Licen
 - Webhook attachments don’t show up in search results.
 - Count of unread mentions are sometimes mixed when switching between multiple teams.
 
+### Contributors
+
+Many thanks to all our contributors. In alphabetical order:
+
+/platform
+
+[asaadmahmoodspin](https://github.com/asaadmahmoodspin), [coreyhulen](https://github.com/coreyhulen), [crspeller](https://github.com/crspeller), [DavidLu1997](https://github.com/DavidLu1997), [eadmund](https://github.com/eadmund), [enahum](https://github.com/enahum), [esethna](https://github.com/esethna), [hmhealey](https://github.com/hmhealey), [jasonblais](https://github.com/jasonblais), [jwilander](https://github.com/jwilander), [lfbrock](https://github.com/lfbrock), [maruTA-bis5](https://github.com/maruTA-bis5), [Rudloff](https://github.com/Rudloff), [samogot](https://github.com/samogot), [yuters](https://github.com/yuters)
```

Comment:
I think Asaad might have changed his GitHub username as his GitHub link is giving a `404 page not found` error
[asaadmahmoodspin](https://github.com/asaadmahmoodspin)